--- origsrc/e_dbus-1.0.1/src/lib/hal/Makefile.am	2011-04-19 01:50:23.000000000 -0500
+++ src/e_dbus-1.0.1/src/lib/hal/Makefile.am	2011-11-04 00:21:27.585124000 -0500
@@ -17,6 +17,6 @@ e_hal_main.c
 libehal_la_LIBADD = \
 $(top_builddir)/src/lib/dbus/libedbus.la
 
-libehal_la_LDFLAGS = -version-info @version_info@ @release_info@
+libehal_la_LDFLAGS = -no-undefined -version-info @version_info@ @release_info@
 
 EXTRA_DIST = e_hal_private.h
--- origsrc/e_dbus-1.0.1/src/lib/notification/Makefile.am	2011-04-19 01:50:23.000000000 -0500
+++ src/e_dbus-1.0.1/src/lib/notification/Makefile.am	2011-11-03 20:30:30.593963400 -0500
@@ -19,6 +19,6 @@ libenotify_la_LIBADD = \
 $(top_builddir)/src/lib/dbus/libedbus.la \
 @EVAS_LIBS@
 
-libenotify_la_LDFLAGS = -version-info @version_info@ @release_info@
+libenotify_la_LDFLAGS = -no-undefined -version-info @version_info@ @release_info@
 
 EXTRA_DIST = e_notify_private.h
